Examples/README.md

```diff
@@ -0,0 +1,13 @@
+This folder contains several examples to show different configurations of the mailserver for different purposes. You can choose from what you need. This list is still limited. Contributions are encouraged.
+
+# Basic functionality
+[basic.md](basic.md) allows sending notifications/alerts, with TLS setups and spoof protection. This can also be a beginner's guide to get a feeling about how the setup works.
+
+# With LDAP functionality
+[ldap.md](ldap.md) shows the setup with LDAP. However, it maybe unclear to a new user what LDAP is and why should one use it. What is the context for it to be useful? Contributions are needed here.
```

Comment:
> Contributions are needed here.

Ah, now I get @erik-wramner comment. instead of asking for contributions in the document I would put this into an issue. Same for the other examples.

Do document what you know, and directly list missing pieces in the readme. Putting questions in a document hardly ever pushes people to contribute. to these questions.